android/tangram/src/com/mapzen/tangram/SceneUpdate.java

```diff
@@ -0,0 +1,24 @@
+package com.mapzen.tangram;
+
+/**
+ * {@code SceneUpdate} Represents a DataStructure to specify a yaml path and the corresponding value for a Scene Update.
+ */
+
+public class SceneUpdate {
+
+    public String path;
+    public String value;
+
+    public SceneUpdate() { this("", ""); }
+
+    public SceneUpdate(String path, String value) {
+        set(path, value);
+    }
+
+    public SceneUpdate set(String p, String v) {
```

Comment:
If we make this public we should be more explicit on the names variable names `p` and `v`. 
I'm not sure we need the default constructor `SceneUpdate()` and this setter since the variable are public.